lib/util.js

```diff
@@ -99,6 +99,7 @@ module.exports = {
   runTests: function(req, res, apiName, grep) {
     mocha = new Mocha();
     mocha.ui('bdd');
+    mocha.timeout(90000);
```

Comment:
This probably will work, but we may have to change it again next time when we run the tests on a different cluster. I think a better way is to allow override this value via query parameters from the request. This way we can actually set the timeout value from the uart tests.

WDYT?